Projects.js

```diff
@@ -1,118 +1,143 @@
-const express=require('express');
-const db=require('./data/helpers/projectModel');
+const express = require('express');
+const db = require('./data/helpers/projectModel');
 const router = express.Router();
 
 
 router.get('/', (req, res) => {
-    db.get()
-    
-     .then(projects => {
-        res.status(200).json(projects)
-          }
-     )
+  db.get()
+
+    .then(projects => {
+      res.status(200).json(projects)
+    }
+    )
     .catch(err => {
-        res.status(500).json({
-          message: 'Server error- Could not retrieve projects.',
-          err
-        })
+      res.status(500).json({
+        message: 'Server error- Could not retrieve projects.',
+        err
       })
-     })
+    })
+})
 
-    router.get('/:id', (req, res) => {
-    const { id } = req.params;
-    db.get(id)
+router.get('/:id', (req, res) => {
+  const { id } = req.params;
+  db.get(id)
 
     .then(project => {
-        if(!id)
-            res.status(404).json({
-            message: "Could not find project with this ID."
+      if (!id)
+        res.status(404).json({
+          message: "Could not find project with this ID."
         })
-        else {
-            res.status(200).json(project)
-        }
+      else {
+        res.status(200).json(project)
+      }
     })
     .catch(err => {
-        res.status(500).json({
-            message: 'Server error- Could not retrieve project.',
-            err
-        })
-        })
-        })
-      
-  
+      res.status(500).json({
+        message: 'Server error- Could not retrieve project.',
+        err
+      })
+    })
+})
 
-router.post('/:id', (req, res) => {
-    const project = req.body;
-    db.insert(project)
+router.get('/:id/actions', (req, res) => {
+  const { id } = req.params;
+  db.getProjectActions(id)
 
-    .then(added => {
-      if (!name || !description) {
+    .then(actions => {
+      if (!id)
         res.status(404).json({
-          message: "Please include name and description"
+          message: "Could not find actions with this ID."
         })
-      } else {
-    res.status(201).json(added)
+      else {
+        res.status(200).json(actions)
```

Comment:
for your if/else statement here.  You're using the `id` as a conditional.  However, `id` will always be truthy as it is coming from `req.params` which will always be there if you get to this route.  Consequently, you will never go into your if statement.  Additionally, think about what `getProjectActions` is returning.  How can you use that?